src/hotspot/share/jvmci/jvmciCompilerToVMInit.cpp

```diff
@@ -249,7 +249,6 @@ JVMCIObjectArray CompilerToVM::initialize_intrinsics(JVMCI_TRAPS) {
   do_bool_flag(UseSHA512Intrinsics)                                        \
   X86_ONLY(do_intx_flag(UseSSE))                                           \
   COMPILER2_PRESENT(do_bool_flag(UseSquareToLenIntrinsic))                 \
-  do_bool_flag(UseStackBanging)                                            \
```

Comment:
Compiler folk will need to verify if this change is okay wrt. upstream Graal code.

Reply:
done